js/everything.js

```diff
@@ -490,13 +510,16 @@ $(function() {
     // Handle form submission
     $("#form").submit(function(event) {
         event.preventDefault();
-
         var formValue = $.trim($("#v").val());
+        var formValueTime = /&t=(\d+)$/g.exec(formValue);
+        if (formValueTime && formValueTime.length > 1) {
+            formValueTime = parseInt(formValueTime[1], 10);
+            formValue = formValue.replace(/(&t=\d*)$/g, "");
```

Comment:
Why `\d*` here, but `\d+` above? Also, why are the parentheses in this regex? I don't think they're needed since we're not capturing a group